src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/SHAHashProvider.Browser.Native.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+using SimpleDigest = Interop.BrowserCrypto.SimpleDigest;
+
+namespace Internal.Cryptography
+{
+    internal sealed class SHANativeHashProvider : HashProvider
+    {
+        private readonly int hashSizeInBytes;
+        private readonly SimpleDigest impl;
+        private MemoryStream? buffer;
```

Comment:
Style: Instance fields should be prefixed with `_`.

```suggestion
        private readonly int _hashSizeInBytes;
        private readonly SimpleDigest _impl;
        private MemoryStream? _buffer;
```